beets/ui/commands.py

```diff
@@ -1479,17 +1479,20 @@ def move_items(lib, dest, query, copy, album, pretend, confirm=False):
             show_path_changes([(obj.path, obj.destination(basedir=dest))
                                for obj in objs])
     else:
-        if confirm:
-            objs = ui.input_select_objects(
-                u'Really %s' % act, objs,
-                lambda o: show_path_changes(
-                    [(o.path, o.destination(basedir=dest))]))
+        if export:
+            util.copy(item.path, item.destination(basedir=dest))
```

Comment:
I noticed something weird. AppVeyor and Travis say that I have 4 failures but when I run the command nosetests locally I get 3 failures and when I run the command python setup.py test locally I get 7 failures. How is that possible? Also in the local tests, no matter what I change in my `if export` statement the failures remain exactly the same.